src/centralized_telescope.rs

```diff
@@ -0,0 +1,419 @@
+//! ALBA's bounded DFS scheme using Blake2b as hash function.
+//! (c.f. Section 3.2.2 of Alba paper)
+
+use crate::utils;
+use blake2::{Blake2s256, Digest};
+use std::{f32::consts::LOG2_E, f64::consts::E};
+
+const DATA_LENGTH: usize = 32;
+const DIGEST_SIZE: usize = 32;
+
+type Element = [u8; DATA_LENGTH];
+type Hash = [u8; DIGEST_SIZE];
+
+/// Setup input parameters
+#[derive(Debug)]
+pub struct Params {
```

Comment:
What if we define `lambda_sec`, `lambda_rel`, and `sec_param` as global variables?

Reply:
I think this relates to the question of the code purpose.
If this  code is meant to be used as reference, we should leave the lambdas as parameters.
However if it is supposed to be a product, our "customers" would be developpers and we should only provide functions with set security. 
What is the code (reference/product) has not been settled, so right now I think it is reasonable to leave the lambdas as variables. We could however also provide functions with set security parameters in case.

Reply:
I have a slight preference for having the security parameters as variables too and maybe suggest some default values though those values will be different for the different applications.